ubports/system-image-upgrader

```diff
@@ -452,6 +453,7 @@ do
                         umount $SYSTEM_PARTITION || true
                         umount $SYSTEM_MOUNTPOINT || true
                         umount /system_root || true
```

Comment:
Do we need /system_root here now? Shouldnt that be deprecated as @amartinz stated?

Reply:
well, devices upgrading to Android 10 can still have it while devices launching with Android 10 usually do not have it.

so we have to keep this to not break devices, which already use it